### PR TITLE
DR5 webpage updates

### PR DESCRIPTION
**Pull request by geordie666**
Rewrite of the DR5 webpages. Main changes include:

- New Data Model where appropriate
- Added `RELEASE=5000` description
- Contents toolbars for quick-links on longer pages
- Additional drop-down menus (e.g., there are now tips and tricks for the viewer)
- `SIMP` has been deprecated by `REX`
- ordering of bands for photometric model fits has changed
- acknowledgment is now consistent with NOAO page
- and many other smaller things

### CODE REVIEW

**Comment by weaverba137** [comment]
@geordie666, that's good, because I'm about to head out of town.  I've already noticed that the gallery link still points to dr4.

**Comment by weaverba137** [comment]
I've skimmed the DR5 pages on the test site.  I haven't found any problems, but my *opinion* is that the table of contents boxes on *e.g.*, http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/output/dr5/description/, are a bit garish.  Would it be possible to change that to a more muted color that matches the look of the rest of the site?  The light gray box on the home page is a good example.

File: CHANGES.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 # legacysurvey Change Log
 
-## 4.0.1 (unreleased)
-
-- No changes yet.
+## 5.0.0 (unreleased)
+
+- Updated to release version for DR4 ([PR#73](https://github.com/legacysurvey/legacysurvey/pull/73)).
```
Comment (weaverba137) [inline CHANGES.md]: Shouldn't this be DR5?

File: pages/bitmask.rst [hunk 2]
```diff
@@ -11,7 +11,7 @@
 .. _`MzLS`: ../mzls
 .. _`DESI`: http://desi.lbl.gov
 .. _`DR3`: ../dr3
-.. _`files page`: ../dr4/files
+.. _`files page`: ../dr5/files
 .. _`legacypipe`: https://github.com/legacysurvey/legacypipe
 
 The ``ccds-`` and ``survey-`` files described on the `files page`_ contain a column ``bitmask``, which is defined as:
```
Comment (weaverba137) [inline pages/bitmask.rst]: I'd like to check the bitmask page, but I can't find which page links to bitmask.rst.

Comment (weaverba137) [reply pages/bitmask.rst]: There was a link to bitmask.rst in dr4/files.rst, but that seems to be missing from dr5/files.rst.

File: CHANGES.md (hunk 1 above)
Comment (geordie666) [reply CHANGES.md]: Yes, thanks. I've changed it.

File: pages/bitmask.rst (hunk 2 above)
Comment (geordie666) [reply pages/bitmask.rst]: Good catch. I've specifically linked this back to the DR4 files page, as you noted, `bitmask` does not appear in the DR5 data model.

**Comment by weaverba137** [comment]
I'm confused by the last commit comment.  Why isn't there a link to the bitmask page on the dr5/files page?

**Comment by geordie666** [comment]
`bitmask` is a column that appeared in the DR4 data model, linked (it
appears) specifically to `ccds-` files created for BASS/MzLS. That column
is not in the DR5 data model, so there is no link to it at `dr5/files`.

However, anyone using the DR4 files will still need a description of
`bitmask`, and hence the need to still link to it from `dr4/files`

**Comment by weaverba137** [comment]
OK, I was looking at your output page, not at the changed file.  Looks good now.

**Comment by geordie666** [comment]
Yeah, sorry, I got lazy with that last push and didn't update my output page. Everything should be synced now.

**Comment by weaverba137** [comment]
@geordie666, What about the sizes of the various directories on dr5/description.rst?  I'm not entirely convinced that any of them are correct. We're still finding stuff that needs to be moved into several of the directories.

**Comment by geordie666** [comment]
Those directory estimates were final as of the point I was informed "all of the relevant files are now in the directory". Basically, as of the time the final files were added to `external` last week. If we change anything (for instance, I don't know that we won't update the gallery at the last moment) then the directory content sizes will, indeed, have to be changed before pushing the website.

If we add new files to "the directory" that aren't in the data model, then the website will need to be further updated with the data model for those files (should they be files that we typically document rather than, e.g., just sub-files in the `logs` directory or some-such).

**Comment by weaverba137** [comment]
Well, unfortunately, that was a bit overoptimistic.  The more I look, the more problems I find.

How, exactly, did the coadd directory shrink in size from 57 TB in DR3 to 18 TB in DR5?

**Comment by geordie666** [comment]
That would probably be a question for @dstndstn. It's possible, even, that the DR3 directory structure was changed *after* the DR3 website went live. I haven't been retroactively checking if earlier data releases have been altered with a new DR. I should check the current DR4 and DR3 directory sizes and update them on the webpage if needed. In other words, if things have been retroactively repackaged, that might explain a change from 57 TB to 18 TB.

**Comment by dstndstn** [comment]
Compression!  DR5 uses lossy fpack compression, DR3 did not.

**Comment by weaverba137** [comment]
Thank you @dstndstn.  May I suggest that we add an asterisk at the bottom of that table then to say that these numbers are estimates, and may change slightly after release, although the actual data files shouldn't change.

**Comment by geordie666** [comment]
@weaverba137: I'll add that to all of the tables for the DR1-5 releases. I'm also running through the DR1-5 directories to check for any major discrepancies in file sizes.

**Pull request merged**